client/faucet/src/main.rs

```diff
@@ -204,7 +208,6 @@ mod tests {
                 )
                 .reply(&filter)
                 .await;
-            assert_eq!(resp.body(), "1"); // 0+1
```

Comment:
something changed? we want to ensure the response number is expected, right?

Reply:
Yeah i'll fix this